doc/sphinx/proof-engine/ltac.rst

```diff
@@ -546,15 +548,13 @@ Failing
 
    This is the always-failing tactic: it does not solve any
    goal. It is useful for defining other tacticals since it can be caught by
-   :tacn:`try`, :tacn:`repeat`, :tacn:`match goal`, or the branching tacticals. The
-   :tacn:`fail` tactic will, however, succeed if all the goals have already been
-   solved.
```

Comment:
Rather than removing this unclear sentence, another possibility would be to explain what is meant more precisely: `all: fail` always fails unless there are not goals left. Similarly, and probably more useful, `easy; fail` will succeed if `easy` succeeds (while `easy; gfail` will always fail). Actually, you have already added this info in the `gfail` description, so maybe it's enough.